contracts/Bridge.sol

```diff
@@ -48,18 +50,21 @@ contract Bridge is RelayerRole, Pausable {
      *   - add/remove relayers
      *   - add/remove tokens that can be bridged
      */
-    constructor(
-        address[] memory board,
-        uint256 initialQuorum,
-        ERC20Safe erc20Safe
-    ) {
+    function initialize(address[] memory board, uint256 initialQuorum, ERC20Safe erc20Safe) public virtual initializer {
```

Comment:
Since this method has been changed from `constructor` to `initialize`, the comment above should also be changed from 

```
     * @dev whoever deploys the contract is the admin
```

To something that reflects the new setup better.

```
     * @dev whoever initializes the contract is the admin
```